app/views/users/show.html.erb

```diff
@@ -0,0 +1,107 @@
+<div class="container-fluid about-page">
+  <%= render :partial => "static_pages/main_nav", :locals=>{:profile => @profile} %>
+  <!--main photo-->
+  <%= render :partial => "static_pages/main_photo", :locals=>{:profile=>@profile} %>
+  
+  <div class="row">
+    <div class="col-md-10 col-md-offset-1">
+      
+      <!--middle navigation-->
+     <%= render :partial => "static_pages/middle_nav", :locals => {:page=>"about", :user => @user} %>
+    <!--about section-->
+    <section class="row about-container">
+    <div class="col-md-12">
+      <div class="row friend-header">
+        <div class="col-md-12">
+        About
+        <% if authorized_user? %>
+         <button type="button" class="btn btn-default btn-small edit-button">
+          <%= link_to "Edit Profile", edit_user_path(@user) %>
+        </button>
+        <% end %>
+      </div>
+
+    </div><!--about section end-->
+    <!--main text-->
+    <div class="row">
+      <!--left side-->
+      <div class="col-md-5" id="information">
+        <div class="basic-information">
+        <h2>Basic Information</h1>
+        <table class="information-table">
+          <tr>
+            <td>Birthday:</td>
+            <td>
+              <%= @profile.birthday.strftime("%B %-d %Y") %>
+            </td>
+          </tr>
+          <tr>
+            <td>College:</td>
+            <% if @profile.college %>
+              <td><%= @profile.college %></td>
+            <% else %>
+              <td>Nothing here yet!</td>
+            <% end %>
+          </tr>
+          <tr>
+            <td>Hometown:</td>
+            <% if @profile.hometown %>
+              <td><%= @profile.hometown %></td>
+            <% else %>
+              <td>Nothing here yet!</td>
```

Comment:
Hm, my hometown in the application is just blank. Wonder why this doesn't work...